pkg/dockerutil/dockerutils.go

```diff
@@ -822,22 +836,8 @@ func CheckDockerVersion(versionConstraint string) error {
 		}
 	}
 
-	constraint, err := semver.NewConstraint(versionConstraint)
-	if err != nil {
-		return err
-	}
-
-	match, errs := constraint.Validate(dockerVersion)
-	if !match {
-		if len(errs) <= 1 {
-			return errs[0]
-		}
-
-		msgs := "\n"
-		for _, err := range errs {
-			msgs = fmt.Sprint(msgs, err, "\n")
-		}
-		return fmt.Errorf("%s", msgs)
+	if !dockerVersions.GreaterThanOrEqualTo(currentAPIVersion, dockerVersionMatrix.APIVersion) {
```

Comment:
This stanza is fine, but it's not obvious in the code what's going on. We're checking the API version, but then recommending the Docker version.

Suggestion below may not work right because of inability to insert tab here.

```suggestion
	// Check against recommended API version, if it fails, suggest the minimum Docker version that relates to supported API
	if !dockerVersions.GreaterThanOrEqualTo(currentAPIVersion, dockerVersionMatrix.APIVersion) {
```

Reply:
Thanks, I do see a tab here (I'm using the [Refined GitHub](https://github.com/refined-github/refined-github) browser extension, which shows tabs and spaces in PRs and adds many other small, fancy features).